backend/routes/events.js

```diff
@@ -1,40 +1,41 @@
 var express = require('express');
 var router = express.Router();
+const passport = require('passport');
 
 //Import Controllers
 const EventController = require('../controllers/EventController');
 
 router.get('/', EventController.getAllEvents);
 router.get('/getEventbyCreator/:id', EventController.getEventbyCreator);
 
-router.post('/', EventController.addEvent);
+router.post('/', passport.authenticate('jwt', {session: false}), EventController.addEvent);
 
 //Media endpoints
 router.post('/:id/media', EventController.addMedia);
```

Comment:
I guess you missed **addMedia**.

Reply:
No, I explaines this in the PR message 😁

Reply:
Sorry for the unattended review :((